libs/react-helpers/src/lib/format/number.ts

```diff
@@ -0,0 +1,27 @@
+import { BigNumber } from 'bignumber.js';
+import memoize from 'lodash/memoize';
+import { getUserLocale } from './utils';
+
+export function addDecimal(value: string, decimals: number): string {
+  if (!decimals) return value;
+  return new BigNumber(value || 0)
+    .dividedBy(Math.pow(10, decimals))
+    .toFixed(decimals);
+}
+
+export function removeDecimal(value: string, decimals: number): string {
```

Comment:
(non-blocking): I suggest the name 'moveDecimal' as that's closer to what the function's doing, and describes another way it could be used

Reply:
Lets discuss tomorrow and make a follow up if necessary